src/components/Login/Login.scss

```
@@ -0,0 +1,95 @@
+@import '/src/styles/variables.scss';
+@import 'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css';
+
+.login {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  background-color: $font-color-grey;
+  padding: 20px 20px 50px 20px;
+  background-color: white;
+  order: 2;
+  .loginIcon {
+    margin: 20px;
+    font-size: $font-size-xl;
+  }
+  .loginTitle {
+    margin-bottom: 30px;
+    font-size: $font-size-xl;
+  }
+  .loginId {
```

```suggestion
  order: 2;
  
  .loginIcon {
    margin: 20px;
    font-size: $font-size-xl;
  }
  
  .loginTitle {
    margin-bottom: 30px;
    font-size: $font-size-xl;
  }
  
  .loginId {
```
가독성을 위해 선택자 사이를 한칸씩 띄워서 작성해주세요!

---

수정 했습니다!